tests/unit/index_runner/helpers.py

```diff
@@ -0,0 +1,48 @@
+from typing import List
+import queue
+import time
+
+
+class MockMessage:
```

Comment:
was Kafka a problem in testing before?

Reply:
ah I see, its only used for the new tests

Reply:
Yeah this is just so we can run super fast unit tests with `scripts/run_unit_tests`

These tests in `tests/unit` will run without any running any docker images and without running any servers so they will be a lot less tedious.